plenum/server/node.py

```diff
@@ -2317,3 +2317,27 @@ def logNodeInfo(self):
                 as logNodeInfoFile:
             logNodeInfoFile.write(json.dumps(self.nodeInfo['data']))
 
+
+    def handleGetTnxReq(self, request: Request, frm: str):
```

Comment:
1) Please move GET_TXN request from sovrin-common to plenum
2) Please add input validation for GET_TXN (have a look at `ClientOperationField`)
3) Add tests for GET_TXN input validation (see `test_client_nym_op.py` and `test_prepare_message.py`)
4) Process GET_TXN in plenum's `processRequest` (move code from sovrin-node)
5) Add test in plenum to send GET_TXN (consider positive and negative test cases)